src/cryptonote_core/cryptonote_tx_utils.h

```diff
@@ -73,17 +73,22 @@ namespace cryptonote
 
   struct tx_destination_entry
   {
+    std::string original;
```

Comment:
What is the purpose to introduce this field?

Reply:
Remembering what the user actually sent to (ie, might be an integrated address, or a openalias address).

Reply:
This field seems to only get assigned values without ever being referenced. Am I missing something?